fcos-config.yaml

```diff
@@ -182,3 +182,19 @@ storage:
       contents:
         inline: |
           podman.crc.testing
+    - path: /etc/tmpfiles.d/podman-docker.conf
+      mode: 0644
+      contents:
+        inline: |
+          L+  /run/docker.sock   -    -    -     -   /run/podman/podman.sock
+    - path: /etc/profile.d/docker-host.sh
+      mode: 0644
+      contents:
+        inline: |
+          export DOCKER_HOST="unix://$(podman info -f "{{.Host.RemoteSocket.Path}}")"
+    - path: /etc/containers/registries.conf.d/999-podman-machine.conf
+      mode: 0644
+      contents:
+        inline: |
+          unqualified-search-registries=["docker.io"]
```

Comment:
My f35 has 
```
$ cat /etc/containers/registries.conf |grep  unqualified-search-registries
unqualified-search-registries = ["registry.fedoraproject.org", "registry.access.redhat.com", "docker.io", "quay.io"]
```
so I don't know why this change is needed. It does not belong in this commit.